index.rst

```diff
@@ -436,7 +436,7 @@ All figures should have a caption and an `Alternative-text (alt-text)`_ statemen
 The motivation and guidance for writing alt-text is provided under
 :ref:`Accessibility considerations<accessibility-considerations>`.
 
-To add a caption and alt-text to an image in rST, use the ``:alt:`` command
+To add a caption and alt-text to an image in rst, use the ``:alt:`` command
```

Comment:
Let's go with RST instead of rst or rST. I've also seen ReST or reST but... it's an acronym so all-caps seems most straightforward.

Reply:
(and RST is used elsewhere in RTN-045)